src/test/java/org/prebid/server/bidder/yeahmobi/YeahmobiBidderTest.java

```diff
@@ -267,6 +254,39 @@ public void makeBidsShouldReturnNativeBidIfNativeIsPresentInRequestImp() throws
                 .containsExactly(BidderBid.of(Bid.builder().impid("123").build(), xNative, "EUR"));
     }
 
+    @Test
+    public void makeBidsShouldReturnVideoInfo() throws JsonProcessingException {
+        // given
+        final BidderCall<BidRequest> httpCall = givenHttpCall(
+                BidRequest.builder()
+                        .imp(singletonList(Imp.builder().id("123").xNative(Native.builder().build()).build()))
+                        .build(),
+                mapper.writeValueAsString(
+                        givenBidResponse(bidBuilder -> bidBuilder
```

Comment:
I suggest hiding `mapper.writeValueAsString` inside the `givenBidResponse` since it's called each test time more like an utility step